apps/payments/next/.env.development

```diff
@@ -28,3 +28,26 @@ GEODB_CONFIG__DB_PATH=../../../libs/shared/geodb/db/cities-db.mmdb
 # GeoDBManagerConfig
 GEODB_MANAGER_CONFIG__LOCATION_OVERRIDE__COUNTRY_CODE=
 GEODB_MANAGER_CONFIG__LOCATION_OVERRIDE__POSTAL_CODE=
+
+# StripeConfig
+STRIPE_CONFIG__API_KEY=11233
+STRIPE_CONFIG__TAX_IDS={}
+
+# PayPal Config
+PAYPAL_CLIENT_CONFIG__SANDBOX=false
+PAYPAL_CLIENT_CONFIG__USER=ASDF
+PAYPAL_CLIENT_CONFIG__PWD=ASDF
+PAYPAL_CLIENT_CONFIG__SIGNATURE=ASDF
+PAYPAL_CLIENT_CONFIG__RETRY_OPTIONS__RETRIES=1
+PAYPAL_CLIENT_CONFIG__RETRY_OPTIONS__MIN_TIMEOUT=1
+PAYPAL_CLIENT_CONFIG__RETRY_OPTIONS__FACTOR=1
+
+# Contentful Config
+CONTENTFUL_CLIENT_CONFIG__CDN_API_URI=https://cdn.contentful.com
+CONTENTFUL_CLIENT_CONFIG__GRAPHQL_API_URI=https://graphql.contentful.com/content/v1
+CONTENTFUL_CLIENT_CONFIG__GRAPHQL_API_KEY=
+CONTENTFUL_CLIENT_CONFIG__GRAPHQL_SPACE_ID=l7gqxxg5i1gg
+CONTENTFUL_CLIENT_CONFIG__GRAPHQL_ENVIRONMENT=dev
+CONTENTFUL_CLIENT_CONFIG__FIRESTORE_CACHE_COLLECTION_NAME=contentfulQueryCacheCollection
+CONTENTFUL_CLIENT_CONFIG__MEM_CACHE_T_T_L=
+CONTENTFUL_CLIENT_CONFIG__FIRESTORE_CACHE_T_T_L=
```

Comment:
Q: Do we have to use `*_T_T_L` suffix or can we change that to `*_TTL` for consistency?

```sh
git grep -Pin "_T_T_L" | cat
apps/payments/next/.env.development:52:CONTENTFUL_CLIENT_CONFIG__MEM_CACHE_T_T_L=
apps/payments/next/.env.development:53:CONTENTFUL_CLIENT_CONFIG__FIRESTORE_CACHE_T_T_L=
apps/payments/next/.env.production:46:CONTENTFUL_CLIENT_CONFIG__MEM_CACHE_T_T_L=
apps/payments/next/.env.production:47:CONTENTFUL_CLIENT_CONFIG__FIRESTORE_CACHE_T_T_L=

git grep -Pin "_TTL" | wc -l # 63
```

Reply:
Unfortunately, we'd need to stick with this suffix for the transformer to change it to the required format of `memCacheTTL`